metazoa/species/Schmidtea/Schmidtea_mediterranea/Schmidtea_mediterranea_GCA_045838265.1cm_about.md

```diff
@@ -0,0 +1,38 @@
+**About *Schmidtea mediterranea***
+-------------------------
+
+The genus Schmidtea comprises four species of freshwater planarians, predominantly found in
+Europe, with some populations also present in north Africa, Asia and introduced in regions of
+America [2, 11]. Among these, Schmidtea mediterranea is the most
+extensively studied species and serves as a well-established model organism for research in
+regeneration and stem cell biology [13, 7].
+
+The life cycle of S. mediterranea features two distinct reproductive strategies observed in
+geographically separated wild populations [10]:
+
+ * Sexual reproduction: Diploid (2n = 8) sexually reproducing populations are found in the
+eastern part of the range of the species, including Corsica, Sardinia, Sicily, and one
+locality in Tunisia. Notably, a triploid (3n = 12) egg-laying population has been identified
+in Sardinia [4].
+
+ * Asexual reproduction by fission: Predominantly diploid (2n = 8) fissiparous forms are
+located in two localities in Catalunya and the Balearic Islands. These individuals exhibit
+a heteromorphic translocation between the first and third chromosomes, visible in
+metaphase spreads, with only one chromosome of each homologous pair affected
+[12, 1]. A triploid (3n = 12) asexual population has also been
+reported in Menorca [12]. Today, the presence of S. mediterranea in
+continental Europe is limited to a few artificial ponds at the "Viver dels Tres Pins" in the
+mountain of Montjuïc, Barcelona [14]. The diploid laboratory strain
+C4 originates from a nearby spring in Montjuïc.
+
+This is an assembly of the sexual laboratory strain S2F2 of S. mediterranea originally collected
+in Sardinia that was extensively inbred over 16 generations to a total of 18 inbreeding
+generations total, including 2 rounds of regeneration-induced self-fertilization of the same
+individual.
+
+For the alternative haplotype see https://metazoa.ensembl.org/Schmidtea_mediterranea_gca045838265.1cm.
+
+Picture credit: Miquel Vila-Farré, Dept. of Tissue Dynamics and Regeneration, Max Planck Institute for Multidisciplinary Sciences
```

Comment:
Just checking: is the licensing for the images ok? Are we allowed to use them?

Reply:
Yes, we even got the emails in case they are needed as well.